examples/encfs/README.md

```diff
@@ -43,11 +43,11 @@ The user needs to upload the blob to the previously generated storage container
 `azcopy copy --blob-type=PageBlob ./generatefs/encfs.img 'https://<storage-container-uri>.blob.core.windows.net/private-container/encfs.img?<SAS_token_to_container_with_write_create_read_permissions>`
 
 ### Security policy generation
-Deploying a confidential container group requires generating a security policy that restricts what containers can run within the container group. There is an az tool available for generation of policies.
+Deploying a confidential container group requires generating a security policy that restricts what containers can run within the container group. There is an az tool available for generating policies. See [here](https://github.com/Azure/azure-cli-extensions/tree/main/src/confcom/azext_confcom#microsoft-azure-cli-confcom-extension-examples) for installing Azure `confcom` CLI extension.  
 
-`az extension add –source https://acccliazext.blob.core.windows.net/confcom/confcom-<latest-version>-py3-none-any.whl -y`
+The ARM template can be used directly to generate a security policy. The following command generates a security policy and automatically injects it into the template. 
 
-`az confcom acipolicygen -i policy-in.json -s policy-out.json -orp`
+    `az confcom acipolicygen -a aci-arm-template.json`
 
 The policy-input file includes two entries: (i) encrypted filesystem sidecar container which whitelists the /encfs.sh as entry point command and the environment variable *EncfsSideCarArgs* used by the script, and (ii) an application container which whitelists a while loop command as entry point command. 

```

Comment:
you may want to make changes to this paragraph given that the policy-input file is not used any more.